src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -735,6 +737,108 @@ describe('MenuBar', () => {
       );
     });
   });
+
+  describe('background click closeout', () => {
+    let fixture: ComponentFixture<MenuBarWithMenus>;
+
+    let menus: CdkMenu[];
+    let triggers: CdkMenuItemTrigger[];
+
+    /** open the attached menu. */
+    const openMenu = () => {
+      triggers[0].toggle();
+      detectChanges();
+    };
+
+    /** set the menus and triggers arrays. */
+    const grabElementsForTesting = () => {
+      menus = fixture.componentInstance.menus.toArray();
+      triggers = fixture.componentInstance.triggers.toArray();
+    };
+
+    /** run change detection and, extract and set the rendered elements. */
+    const detectChanges = () => {
+      fixture.detectChanges();
+      grabElementsForTesting();
+    };
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkMenuModule],
+        declarations: [MenuBarWithMenus],
+      }).compileComponents();
+    }));
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(MenuBarWithMenus);
+      detectChanges();
+    });
+
+    it('should close out all open menus when clicked outside the menu tree', () => {
+      openMenu();
+      expect(menus.length).toBe(1);
+
+      fixture.debugElement.query(By.css('#container')).nativeElement.click();
+      detectChanges();
+
+      expect(menus.length).toBe(0);
+    });
+
+    it('should not close open menus when clicking on a menu group', () => {
+      openMenu();
+      expect(menus.length).toBe(1);
+
+      const menuGroups = fixture.debugElement.queryAll(By.directive(CdkMenuGroup));
+      menuGroups[0].nativeElement.click();
+      menuGroups[1].nativeElement.click();
```

Comment:
Just one may be fine. (With multiple clicks, future readers might wonder if there's a particular reason you clicked multiple times.)